modules/ROOT/pages/changelogs.adoc

```diff
@@ -1,367 +1,32 @@
+Using the error messages programmatically is deprecated and might change in a future release.
+Therefore, parsing the error messages or the status descriptions and incorporating them into scripts is not recommended.
+If you need to handle errors programmatically, please use the GQLSTATUS error codes, for example, `22NC5`.
 
-| 01N82
-| Neo.ClientNotification.Cluster.ServerNotAvailable
+== 2025.01
 
-| 03N85
-| Neo.ClientNotification.Cluster.ServerCaughtUp
+* The diagnostic record also contains a `_status_parameters` field, which represents a map that contains all variable parts of the status description, such as labels, database names, Cypher clauses, etc.
+This field is used to provide more detailed information about the error.
 
-|===
+* A new JSON template is available for the query log, which is the default set in _server_logs.xml_. +
+For more details, see the link:https://neo4j.com/docs/operations-manual/current/monitoring/logging/#_json_format_log_entries[Operations manual -> Monitoring -> Logging].
 
 
-//Starting from 2025.01, when using Cypher 25, queries using `WAIT` return the notifications listed above instead of result rows.
-//The behaviour for Cypher 5 remains unchanged.
+ifndef::backend-pdf[]
+[discrete.glossary]
+== Glossary
 
-== Neo4j 5.26
-
-**New:**
-
-[options="header", cols="<1m,<1"]
-|===
-
-| GQLSTATUS
-| Neo4j code
-
-| 01N72
-| Neo.ClientNotification.Statement.InsecureProtocol
-
-| 42I50
-| Neo.ClientError.Schema.TokenLengthError
-
-|===
-
-Starting from 5.26, Neo4j errors sent over drivers include new fields,
-such as GQLSTATUS code, StatusDescription, and DiagnosticRecord.
-For more information, see xref:errors/index.adoc#gqlstatus-error-object[GQL-status error object].
-
-
-== Neo4j 5.25
-
-Starting from 5.25, the query log includes the GQL error information under the JSON object `errorInfo`.
-For more information, see link:https://neo4j.com/docs/operations-manual/current/monitoring/logging/#gql-error-information[Operations Manual -> GQL error information].
-
-== Neo4j 5.24
-
-**New:**
-
-[options="header", cols="<1m,<1"]
-|===
-
-| GQLSTATUS
-| Neo4j code
-
-| 00N72
-| Neo.ClientNotification.Security.AuthProviderNotDefined
-
-| 01G11
-| Neo.ClientNotification.Statement.AggregationSkippedNull
-
-| 01N71
-| Neo.ClientNotification.Security.ExternalAuthNotEnabled
-
-| 03N61
-| Neo.ClientNotification.Statement.RedundantOptionalProcedure
-
-| 03N62
-| Neo.ClientNotification.Statement.RedundantOptionalSubquery
-
-| N/A
-| Neo.TransientError.Invalid.ResourceExhaustion
-
-|===
-
-== Neo4j 5.23
-
-From version 5.23, Neo4j has a new GqlStatusObject API in addition to the existing Notification API.
-
-The GqlStatusObject API provides information about the status of a Cypher query or command execution in compliance with the GQL standard.
-It includes GQLSTATUS code, StatusDescription, and DiagnosticRecord.
-For more information, see xref:notifications/index.adoc#gqlstatus-notification-object[GQL-status notification object].
```

Comment:
I think this would still be interesting to keep as it is a change to the framework

Reply:
Fair point, I did not think about this